### PR TITLE
[npm] Switch to using our own sql formatter

### PR DESCRIPTION
**Pull request by romainr**

### CODE REVIEW

File: .npmrc [hunk 1]
```diff
@@ -0,0 +1 @@
+@gethue:registry=https://npm.pkg.github.com/
```
Comment (JohanAhlen) [inline .npmrc:1]: This needed?

File: package.json [hunk 2]
```diff
@@ -79,7 +79,7 @@
     "selectize": "0.12.6",
     "selectize-plugin-clear": "0.0.3",
     "sprintf-js": "1.1.2",
-    "sql-formatter": "^4.0.2",
+    "@gethue/sql-formatter": "4.0.2",
```
Comment (JohanAhlen) [inline package.json:82]: also package-lock.json?

File: .npmrc (hunk 1 above)
Comment (romainr) [reply .npmrc:1]: Yes, but no bueno still, not sure what is missing. The idea is that this package is published on the GH npm, not the default one

Comment (JohanAhlen) [reply .npmrc:1]: For now bueno, can look into it later.

**Review by JohanAhlen — APPROVED** [review]

**Comment by romainr** [comment]
One potential issue: downstream would need a GH PAT too..

**Comment by romainr** [comment]
Switched to just traditional npm public package (not github) https://www.npmjs.com/package/@gethue/sql-formatter

**Pull request merged**